Automattic/wp-calypso
# Subscriptions Management: Render site subscription details page in reader

Closes #81270

## Proposed Changes

![Screen Capture on 2023-09-05 at 14-05-06](https://github.com/Automattic/wp-calypso/assets/2019970/b277efe2-61fd-4089-be08-3f70700fd0fd)


* Display the site subscription details when clicking on a WPCOM site subscription title on https://wordpress.com/read/subscriptions (behind a feature flag)
* Display the site subscription details at the following URL: `https://wordpress.com/read/subscriptions/${subscriptionId}`
* For non-wpcom site subscriptions still point to: `/read/feeds/${feedId}`

## Testing Instructions



* Go to http://calypso.localhost:3000/read/subscriptions
* Chose any of your WPCOM site subscriptions, and click on its title
* You should expect the new site subscription details page to open
* Play around with the page and also test going back to the list of site subscriptions by clicking on the "Manage all subscriptions" button

## Pre-merge Checklist
